src/SprykerEco/Yves/Unzer/Form/MarketplaceCreditCardSubForm.php

```diff
@@ -0,0 +1,42 @@
+<?php
+
+/**
+ * MIT License
+ * For full license information, please view the LICENSE file that was distributed with this source code.
+ */
+
+namespace SprykerEco\Yves\Unzer\Form;
+
+use SprykerEco\Shared\Unzer\UnzerConfig;
+
+class MarketplaceCreditCardSubForm extends CreditCardSubForm
+{
+    /**
+     * @var string
+     */
+    protected const PAYMENT_METHOD_NAME = 'marketplace_credit_card';
```

Comment:
Would it make sense to have this value in config too? I mean something like PAYMENT_METHOD_TEMPLATE_MARKETPLACE_CREDIT_CARD?

Reply:
I don't think we need it, since this value is used only for this form.

Reply:
Got your point, but do we have to define additional constant? So we have same naming everywhere or not?